features/pull_request.feature

```diff
@@ -357,28 +361,6 @@ Feature: hub pull-request
     Then the output should contain exactly "https://github.com/mislav/coral/pull/12\n"
     And the file ".git/PULLREQ_EDITMSG" should not exist
 
-  Scenario: Text editor with custom commentchar
-    Given git "core.commentchar" is set to "/"
```

Comment:
Could this scenario stay active, but have it updated to start passing?

The `commentchar` value will dictate the first character of the scissors line.